client/tests/webdriver/baseSpecs/createNewPerson.spec.js

```diff
@@ -182,11 +182,6 @@ describe("Create new Person form page", () => {
         await CreatePerson.getEmailAddress()
       ).setValue(VALID_PERSON_ADVISOR.emailAddress)
       await (await CreatePerson.getLastName()).click()
-      let errorMessage = await browser.$(
-        "input#emailAddress + div.invalid-feedback"
-      )
-      // element should *not* be visible!
-      await errorMessage.waitForDisplayed({ timeout: 1000, reverse: true })
```

Comment:
You should move this down (after line 205), to assert that you _don't_ get the error message anymore.